engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -3293,6 +3318,51 @@ namespace dmGameSystem
         return dmGameObject::RESULT_OK;
     }
 
+
+    dmGameObject::Result CompGuiRegisterSetPropertyFn(dmhash_t property_hash, CompGuiPropertySetterFn setter)
+    {
+        if (g_CompGuiPropertySetters.Full())
+        {
+            uint32_t capacity = g_CompGuiPropertySetters.Capacity();
+            capacity = capacity > 0 ? capacity * 2 : 16;
+            g_CompGuiPropertySetters.SetCapacity(capacity * 2, capacity);
```

Comment:
We recently added an `dmHashTable::OffsetCapacity()` to help with this, to help avoid accidentally creating a single linked list.
It's not the case here, but more-or-less the case with the reverse hash table.
So to help with "documentation", I now prefer using the OffsetCapacity() function.